pkgs/top-level/all-packages.nix

```diff
@@ -21608,7 +21608,7 @@ in
     inherit (darwin.apple_sdk.frameworks) CoreServices;
   };
 
-  zoom-us = libsForQt59.callPackage ../applications/networking/instant-messengers/zoom-us { };
+  zoom-us = libsForQt5.callPackage ../applications/networking/instant-messengers/zoom-us { };
```

Comment:
I think this was done with 5.9 originally because that's what zoom bundles with. CHanging it to 5.12 could cause ABI incompatibilities.

Reply:
On master it uses latest qt... This on stable also, so the version isn't going to change.

We should probably pin it to what it's bundled on master (check this to).

Reply:
Qt has an ABI stability guarantee. It's always forward compatible.